pkg/apis/openebs.io/v1alpha1/cstor_volume_claim.go

```diff
@@ -95,6 +95,8 @@ type CStorVolumeClaimStatus struct {
 	// Capacity the actual resources of the underlying volume.
 	Capacity   corev1.ResourceList         `json:"capacity,omitempty"`
 	Conditions []CStorVolumeClaimCondition `json:"condition,omitempty"`
+	// PoolInfo represents current pool names where volume replicas exists
+	PoolInfo []string `json:"poolInfo"`
```

Comment:
why this is different from Spec? do you see a need for this as well same as []ReplicaPoolInfo instead of []string?

Reply:
IMO spec can have different things like `name` & `uid`.. then I think status should strictly have only one information which conveys intent.